templates/theme/styles/base.less

```diff
@@ -1,11 +1,6 @@
 @import "syntax-variables";
 
-.editor-colors {
```

Comment:
The `.editor-colors` selector is deprecated, so it's definitely right to remove this.